lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1129,6 +1131,31 @@ def get_response(self, course):
             follow=True
         )
 
+    # TODO: LEARNER-71: Delete entire TestAccordionDueDate class
```

Comment:
Reviewers: I know this is an odd way to handle this, but it worked and its temporary, so I think we should leave it.   

NOTE: I had tried adding the decorator on the original test methods in BaseDueDataTests, and passed in a class variable for 'active', but the tests failed and the debugger never went into the decorator code.